running/safari.md

```diff
@@ -2,9 +2,9 @@
 title: Use Cockpit with Safari on an iPhone or iPad
 ---
 
-Safari on iOS and iPadOS 14.5 has experimental support for secure WebSocket sessions, which Cockpit needs to function.
+Safari's default WebSocket framework on iOS, iPadOS, and ARM-based Macs does not support the WebSocket Secure sessions which Cockpit needs to function when using a self-signed or otherwise invalid certificate, even when they are manually marked as trusted. [Installing a valid certificate]({{ site.baseurl }}/guide/latest/guide.html) should resolve this issue.
```

Comment:
I suggest to directly link to /https.html instead, i.e. https://cockpit-project.org/guide/latest/https.html -- then it's much easier to find.

Seems a bit odd that I don't see this second "Update running.." commit on your [preview](https://garrett.github.io/cockpit-project.github.io/running/safari.html)? Did you have to sync this manually, as this is not coming from your fork's main branch?

Reply:
Oh, this is in a branch, so I have to force-push to main. I'll squash the changes in, change the link and force-push now.